.gitattributes

```diff
@@ -0,0 +1,63 @@
+# Code
+*.py text eol=lf diff=python
+*.html text diff=html
+*.js text
```

Comment:
No specific support for JS?

Reply:
as explained there: https://tekin.co.uk/2020/10/better-git-diff-output-for-ruby-python-elixir-and-more
there is currently not diff specific for javascript in latest git : https://github.com/git/git/blob/master/userdiff.c

Reply:
note: it's seems possible to had custom one in git.